sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -227,24 +248,24 @@ def _download_chunk(self, chunk_start, chunk_end):
 
 
 class _ChunkIterator(object):
-    """Async iterator for chunks in blob download stream."""
+    """Iterator for chunks in blob download stream."""
 
-    def __init__(self, size, content, downloader, chunk_size):
+    def __init__(self, size: int, content: bytes, downloader: Optional[_ChunkDownloader], chunk_size: int) -> None:
         self.size = size
         self._chunk_size = chunk_size
         self._current_content = content
         self._iter_downloader = downloader
-        self._iter_chunks = None
+        self._iter_chunks: Optional[Generator[int, None, None]] = None
         self._complete = (size == 0)
 
-    def __len__(self):
+    def __len__(self) -> int:
         return self.size
 
     def __iter__(self):
```

Comment:
Type hint here? I think its `-> Iterator[bytes]`. Then it might complain that `self` does not have that type which I guess we can ignore? If there is another error, post here and we can look.